test/e2e/run_test.go

```diff
@@ -305,10 +304,6 @@ var _ = Describe("Podman run", func() {
 	})
 
 	It("podman run notify_socket", func() {
-		host := GetHostDistributionInfo()
-		if host.Distribution != "rhel" && host.Distribution != "centos" && host.Distribution != "fedora" {
-			Skip("this test requires a working runc")
-		}
```

Comment:
this works only on the Fedora/RHEL package, as we are carrying an extra patch.

upstream doesn't support it yet: https://github.com/opencontainers/runc/pull/1807

Reply:
Test passes, though?

Reply:
I see it is failing on Ubuntu

Reply:
Hm. Fair enough. I'll disable it again and focus on the other tests, which seems to be just changed mount opts.